vm/systemSmartContracts/delegation.go

```diff
@@ -1966,6 +1980,85 @@ func (d *delegation) getClaimableRewards(args *vmcommon.ContractCallInput) vmcom
 	return vmcommon.Ok
 }
 
+func (d *delegation) isDelegator(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
```

Comment:
Please add these functions in the Jira task list so we can have unit tests for them.

Reply:
Bogdan already did it and opened PR

Reply:
We can easily forget about them if the PRs overlap

Reply:
the PR with unit tests is directed to this branch. We are safe

Reply:
You forgot to test a function